src/main/java/core/basesyntax/service/filesoperation/ReadDataFromFileImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.filesoperation;
+
+import core.basesyntax.service.parsing.ParsingDataService;
+import core.basesyntax.service.parsing.ParsingDataServiceImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReadDataFromFileImpl implements ReadDataFromFile {
+    private static final String ERROR_READ_MESSAGE = "Can't find file by path: ";
```

Comment:
This is unnecessary in my opinion, in this case it will be necessary to return to the announcement in order to read the text of the error.